src/generators/minecraft-axolotl-character/MinecraftAxolotlCharacterGenerator.res

```diff
@@ -0,0 +1,1014 @@
+      //Left Side
+      "Skin",
+      (8, 36, 4, 11),
+      (411, 524, 8, 24),
+      (),
+    )
+    Generator.drawTexture(
+      //Left Foot
+      "Skin",
+      (8, 47, 4, 1),
+      (403, 548, 24, 16),
+      (),
+    )
+
+    // Left Leg
+    Generator.drawTexture(
+      //Right Side
+      "Skin",
+      (0, 52, 4, 11),
+      (445, 524, 8, 24),
+      (),
+    )
+    Generator.drawTexture(
+      //Right Foot
+      "Skin",
+      (0, 63, 4, 1),
+      (437, 548, 24, 16),
+      (),
+    )
+    Generator.drawTexture(
+      //Left Side
+      "Skin",
+      (8, 52, 4, 11),
+      (469, 524, 8, 24),
+      (),
+    )
+    Generator.drawTexture(
+      //Left Foot
+      "Skin",
+      (8, 63, 4, 1),
+      (461, 548, 24, 16),
+      (),
+    )
+
+    //Fins
+    //Tail
+    Generator.drawTexture(
+      //Left Side
+      "Skin",
+      (35, 36, 1, 12),
+      (258, 533, 40, 168),
+      (),
+    )
+    Generator.drawTexture(
+      //Left Side
+      "Skin",
+      (36, 36, 1, 12),
+      (298, 533, 40, 168),
+      (),
+    )
+    //Head
+    Generator.drawTexture(
+      //Top Left Fin's Backside
+      "Skin",
+      (41, 1, 2, 1),
+      (273, 7, 24, 24),
+      ~flip=#Horizontal,
+      (),
+    )
+    Generator.drawTexture(
+      //Top Left Fin's Frontside
+      "Skin",
+      (41, 2, 2, 1),
+      (249, 7, 24, 24),
+      (),
+    )
+    Generator.drawTexture(
+      //Top Right Fin's Backside
+      "Skin",
+      (45, 1, 2, 1),
+      (299, 7, 24, 24),
+      ~flip=#Horizontal,
+      (),
+    )
+    Generator.drawTexture(
+      //Top Right Fin's Frontside
+      "Skin",
+      (45, 2, 2, 1),
+      (323, 7, 24, 24),
+      (),
+    )
+    Generator.drawTexture(
+      //Middle Left Fin's Backside
+      "Skin",
+      (33, 8, 1, 3),
+      (268, 41, 24, 32),
+      (),
+    )
+    Generator.drawTexture(
+      //Middle Left Fin's Frontside
+      "Skin",
+      (34, 8, 1, 3),
+      (268, 73, 24, 32),
+      (),
+    )
+    Generator.drawTexture(
+      //Middle Right Fin's Backside
+      "Skin",
+      (54, 8, 1, 3),
+      (304, 41, 24, 32),
+      (),
+    )
+    Generator.drawTexture(
+      //Middle Right Fin's Frontside
+      "Skin",
+      (53, 8, 1, 3),
+      (304, 73, 24, 32),
+      (),
+    )
+    Generator.drawTexture(
+      //Bottom Left Fin's Backside
+      "Skin",
+      (34, 14, 1, 2),
+      (268, 110, 24, 24),
+      (),
+    )
+    Generator.drawTexture(
+      //Bottom Left Fin's Frontside
+      "Skin",
+      (33, 14, 1, 2),
+      (268, 134, 24, 32),
+      (),
+    )
+    Generator.drawTexture(
+      //Bottom Right Fin's Backside
+      "Skin",
+      (53, 14, 1, 2),
+      (304, 110, 24, 24),
+      (),
+    )
+    Generator.drawTexture(
+      //Bottom Right Fin's Frontside
+      "Skin",
+      (54, 14, 1, 2),
+      (304, 134, 24, 24),
+      (),
+    )
+  }
+
+  //Axolotl Eyes
+  if showEyes {
+    Generator.drawTexture("Skin", (9, 12, 6, 2), (221, 229, 80, 16), ())
```

Comment:
The eyes will be in different positions for different skins so this might not work very well.

Here's a suggestion. The Axolotl face looks like it's stretched horizontally, so what about applying that idea to the character face? I.e. just stretch it horizintally.

For example, here's some code:

```
if showEyes {
  let ox = 229
  let oy = 214
  let stretch = 16
  Generator.drawTexture(
    "Skin",
    steve.base.head.front,
    (ox - stretch, oy, 64 + stretch * 2, 40),
    (),
  )
  if showOverlay {
    Generator.drawTexture(
      "Skin",
      steve.overlay.head.front,
      (ox - stretch, oy, 64 + stretch * 2, 40),
      (),
    )
  }
}
````

Also, we can still use the same photo you have already made since it's just the texture we're changing a bit.

How does that sound?

Reply:
sounds good! I was trying to think of how to go about that but didn't come up with a good solution.